validations.rb

```diff
@@ -0,0 +1,31 @@
+module Validations
```

Comment:
After seeing how you implemented this validations module, I think you could've moved these methods into the BankAccount. It makes more sense to have them as `bank_account_instance.validate_withdrawal`, as withdrawal is an action on a BankAccount instance

You can also keep them separate, but find better naming (like AccountValidations), note it as a class and use instance methods, like:
```ruby
class AccountValidations # or BankValidations
  class << self
    def validate_amount(amount)
      # logic
    end
  end
end

# and call it using
AccountValidations.validate_amount(10)
```